### PR TITLE
Drop url loader

### LINKED ISSUE #888: Drop url-loader
We should drop `url-loader` from webpack and stick to the `file-loader`. Url-loader inlines the SVG images in the CSS file outputted by Webpack which makes this file to have 1.6 MB size.

### PR DESCRIPTION
**Pull request by benekex2**
Fix #888

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ var config = {
       },
       {
         test: /\.(eot|otf|png|svg|jpg|ttf|woff|woff2)(\?v=[0-9.]+)?$/,
-        loader: 'url?name=[name].[hash].[ext]',
+        loader: 'file?name=[name].[hash].[ext]&publicPath=/static/assets/',
```
Comment (patrys) [inline webpack.config.js]: This can be set in global `output` config.

**Comment by krzysztofwolski** [comment]
@patrys @elwoodxblues How does it looks?

**Comment by patrys** [comment]
Looks ok but will probably break production use until we split webpack settings. Anyone willing to test this?

File: circle.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ dependencies:
     - yarn
     - pip install -r requirements.txt
   override:
-    - yarn run build-assets --production
+    - yarn run build-assets:prod --production
```
Comment (patrys) [inline circle.yml]: >`...:prod --production`

We need to go deeper.

File: webpack.config.js [hunk 3]
```diff
@@ -72,7 +86,7 @@ var config = {
       },
       {
         test: /\.(eot|otf|png|svg|jpg|ttf|woff|woff2)(\?v=[0-9.]+)?$/,
-        loader: 'url?name=[name].[hash].[ext]',
+        loader: 'file?name=[name].[hash].[ext]',
```
Comment (patrys) [inline webpack.config.js]: Shouldn't this depend on env?

**Pull request merged**